errgroup/errgroup_test.go

```diff
@@ -1,3 +1,7 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Do we need to copy this header? We don't have a LICENSE file that matches the one from Go.

Reply:
Oh yeah.. good point.  Pushing a commit that will hopefully cross our Ts and dot our Is.